<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mystic Square</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../annexes_projet_2023/240px-15-puzzle.png</normaloff>../annexes_projet_2023/240px-15-puzzle.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="Button_new_game">
        <property name="text">
         <string>New game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_load">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Number of moves :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu0">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCr_er_une_nouvelle_partie"/>
    <addaction name="actionCharger_une_partie"/>
    <addaction name="actionSave_game"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuBackground">
    <property name="title">
     <string>Background</string>
    </property>
    <addaction name="actionOriginal"/>
    <addaction name="actionWood"/>
    <addaction name="actionTree"/>
    <addaction name="actionNetwork"/>
   </widget>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>ðŸ›ˆ</string>
    </property>
    <addaction name="action_About_Qt"/>
    <addaction name="action_About_Mystic_Square"/>
   </widget>
   <addaction name="menu0"/>
   <addaction name="menuBackground"/>
   <addaction name="menu1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCr_er_une_nouvelle_partie">
   <property name="text">
    <string>New game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCharger_une_partie">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_game">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>ðŸ›ˆ About Qt</string>
   </property>
  </action>
  <action name="action_About_Mystic_Square">
   <property name="text">
    <string>ðŸ›ˆ About Mystic Square</string>
   </property>
  </action>
  <action name="actionOriginal">
   <property name="text">
    <string>Original</string>
   </property>
  </action>
  <action name="actionWood">
   <property name="text">
    <string>Wood</string>
   </property>
  </action>
  <action name="actionTree">
   <property name="text">
    <string>Tree</string>
   </property>
  </action>
  <action name="actionNetwork">
   <property name="text">
    <string>Network</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
